### ISSUE TITLE
Snapshots from https://freemint.github.io/#snapshots should be m68k-mint, not linux executables

### ISSUE THREAD

**Issue by mikrosk** [freemint/mintbin#4]
Right now https://tho-otto.de/snapshots/mintbin contains 64-bit Linux ELF binaries.

**Comment by th-otto** [comment]
Yes, that's correct, because those snapshots are used in the CI scripts of other projects. I deliberately changed that in https://github.com/freemint/mintbin/commit/39fafe242088d7820ad928d0d301c9b711b2a085

**Comment by mikrosk** [comment]
I'm not against providing binaries for CI (we did/do that for gcc/binutils as well) but currently there's no way to install a working system from scratch on aranym. Could you maybe upload to your snapshots directory both versions?

**Comment by th-otto** [comment]
Yes, but that requires some changes to build script and also to the deploy script (the deploy script currently only handles a single archive)

Also, in the old version only a single version of the native binaries was built.

But you can do such changes yourself. Just keep in mind that for the linux binaries, symlinks are set up that point to the latest versions. Only those are used in the CI builds.

Also keep in mind that i have already prepared the script to use both toolchains (a.out & mintelf)
